jobs/config.json

```diff
@@ -1970,7 +2165,8 @@
       "--timeout=180m",
       "--extract=ci/latest-1.7",
       "--check-leaked-resources",
-      "--test_args=--ginkgo.focus=\\[Feature:GPU\\]"
+      "--test_args=--ginkgo.focus=\\[Feature:GPU\\]",
+      "--gcp-project=k8s-jkns-e2e-gce-gpus-1-7"
```

Comment:
Also, we can delete `"--env-file=jobs/ci-kubernetes-e2e-gce-gpu-1-7.env",` from this block, now that the file doesn't is empty.